src/js/hca/components/HealthCareApp.jsx

```diff
@@ -129,23 +131,38 @@ class HealthCareApp extends React.Component {
     e.preventDefault();
     const veteran = this.props.data;
     const path = this.props.location.pathname;
+    let apiUrl = `${window.VetsGov.api.url}/api/hca/v1/application`;
+    let formSubmissionId;
+    let timestamp;
+    const testBuild = __BUILDTYPE__ === 'development' || __BUILDTYPE__ === 'staging';
+    const submissionPost = {
+      method: 'POST',
+      headers: {
+        Accept: 'application/json',
+        'Content-Type': 'application/json'
+      },
+      timeout: 10000, // 10 seconds
+      body: veteranToApplication(veteran)
+    };
 
     window.dataLayer.push({ event: 'submit-button-clicked' });
     const formIsValid = validations.isValidForm(veteran);
 
+    if (testBuild) {
```

Comment:
I think I'd put a comment here about why this is necessary right now.

Also, and it may not be worth it at this point, we do have the ability to create "feature flags" now in the webpack config that can be used for this sort of thing.